sites/hashdev/src/_pages/blog/2_open-source.mdx

```diff
@@ -16,34 +16,32 @@ Depending on the code in question, it'll typically be released under one of thre
 1.  **Commercial entities can help open-source projects be more effective.** Wolfram argue that "hard cases and boring stuff need to get done too" (4), essentially implying that open-source maintainers and coders can't be trusted to knuckle down and focus on important maintainability, or boring background work required to make new technologies impactful and successful. Not only is this patronizing to the thousands of open-source developers who care passionately about the products they build, but it ignores the possibility of commercial entities advancing OSS projects and acting with similar motivations to any other private company. "Our developers work for you, not just themselves" (6) relies upon the same misrepresentation, while "Paid software offers an open quid pro quo" (10) mischaracterizes and ignores the various business models that allow OSS to be both free at the point of access, and profitable (e.g. through the provision of paid complements, such as the famous support services provided by _Red Hat_ which justified their $34 billion sale to _IBM_ in 2019).
-1.  **Huge technological advances occur in the open.** Wolfram's claim that "Open source doesn’t bring major tech innovation to market" (9) can be disproven by a thousand major projects, and the entirety of the modern web. We won't list the many projects here that prove this statement's desperate vacuity.
-1.  **There are many long-term financing options and sustainable business models available to open-source projects.** According to Wolfram, "it takes steady income to sustain long-term R&D" (11), which is generally true (unless you happen to be sat on a large nest-egg already). However, many OSS projects are able to command a stable income, making this again _not an argument against open source at all_. In fact, there are various OSS companies that are far more profitable than Wolfram. Since the advent of debt in approximately 3500 BC it's been possible to operate without monetizing upfront -- in promise or expectation of future cash-flows -- and modern financing in the form of venture capital is just one form of this that has led to an explosion in OSS companies. To ignore the last five millenia of human experience and financial innovation is disingenuous.
+1.  **Huge technological advances occur in the open.** Wolfram's ninth claim, that "Open source doesn’t bring major tech innovation to market", can be disproven by a thousand major open projects, and the entirety of the modern web is built atop OSS. To pick on one such project, Linux (whose open Git repositories now contain some ~28 million lines of code) powers 100% of the world’s top 500 supercomputers, 96.3% of the top 1 million web servers, and 23 of the top 25 websites in the world.
```

Comment:
Not vital, but if we're aiming for maximum clarity I'd avoid parenthetical statements of this length, because it's easy to forget what came before them once you get to the end of it.
```suggestion
1.  **Huge technological advances occur in the open.** Wolfram's ninth claim, that "Open source doesn’t bring major tech innovation to market", can be disproven by a thousand major open projects, and the entirety of the modern web is built atop OSS. To pick one such project, Linux powers 100% of the world’s top 500 supercomputers, 96.3% of the top 1 million web servers, and 23 of the top 25 websites in the world – built on open Git repositories now containing some ~28 million lines of code.
```